Assets/Scripts/VolumeHandler.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Audio;
+using UnityEngine.UI;
+
+namespace PeppaSquad.Settings
+{
+    
+public class VolumeHandler : MonoBehaviour
+{
+public AudioMixer MusicMixer;
+public Slider MusicSlider;
+public Slider SfxSlider;
+public Toggle MuteToggle;
+
+public void SetLevelMusic(){
+    
+    MusicMixer.SetFloat("MusicVol", Mathf.Log10(MusicSlider.value)*20);
```

Comment:
De strings die hier worden gebruikt zouden in variabele horen en zouden zelfs constant kunnen zijn.